src/main/java/org/terasology/launcher/util/FileUtils.java

```diff
@@ -94,7 +95,7 @@ public static boolean extractZipTo(final Path archive, final Path outputLocation
             if (Files.notExists(outputLocation)) {
                 Files.createDirectories(outputLocation);
             }
-            try (FileSystem fileSystem = FileSystems.newFileSystem(archive, null)) {
+            try (FileSystem fileSystem = FileSystems.newFileSystem(archive, ((ClassLoader) null))) {
```

Comment:
This had me scratching my head for a minute before I checked the commit messages. Then that not only explained it but included a link to the relevant spot in the release notes. :+1: